offload/cmake/caches/AMDGPUBot.cmake

```
@@ -0,0 +1,14 @@
+set(LLVM_ENABLE_PROJECTS "clang;lld" CACHE STRING "")
+set(LLVM_ENABLE_RUNTIMES "compiler-rt;openmp;offload" CACHE STRING "")
+set(LLVM_ENABLE_PER_TARGET_RUNTIME_DIR ON CACHE BOOL "")
+set(LLVM_ENABLE_ASSERTIONS ON CACHE BOOL "")
+set(LLVM_LIT_ARGS "-vv --show-unsupported --timeout 100 --show-xfail -j 64" CACHE STRING "")
```

```suggestion
set(LLVM_LIT_ARGS "-v --show-unsupported --timeout 100 --show-xfail -j 64" CACHE STRING "")
```
`-vv` is documented as a "Deprecated alias for -v".